app/agents/chain_of_thoughts.py

```diff
@@ -42,14 +44,18 @@ def get_llm():
     return llm
  
 class ChainOfThoughtsAgent(BaseAgent):
-    def __init__(self, guidance, retriever, num_iter=3):
+  
+    def __init__(self, guidance, llama_model, llama_model2):
         self.guidance = guidance
-        self.num_iter = num_iter
-        self.prompt_template = agent_template
-        if TEST_MODE =="ON":
-            self.llm = get_llm()
-            self.retriever = ingest_file(TEST_FILE)
+        self.llama_model = llama_model
```

Comment:
we should add a comment above each model to help readers understand which does what

Reply:
Indeed, it's not very self explanatory right now! Added :)